token_exchange/models.py

```diff
@@ -20,3 +20,6 @@ class TokenExchangeConfiguration(models.Model):
 
     class Meta:
         verbose_name = _("Token exchange plugin configuration")
+
+    def __str__(self) -> str:
+        return self.service.label
```

Comment:
In theory this can break on unsaved instances:

```py
str(TokenExchangeConfiguration())
```

this will crash with `TypeError` because `None` does not have an attribute `label`. This is notably a problem when Sentry is trying to grab a string representation of variables when another exception happens, so the impact is rather annoying if it happens.

It also causes performance issues if the `service` is not `select_related` because it spawns a query for every item in the list, so you get the dreaded N + 1 problem.

Wouldn't it perhaps be simpler to just add a `label` field to the `TokenExchangeConfiguration` model itself?